.github/workflows/release_prep.sh

```diff
@@ -22,9 +30,9 @@ Follow https://github.com/chainguard-dev/rules_apko/blob/main/docs/initial-setup
 EOF
 
 cat << EOF
-## Using Bzlmod
```

Comment:
I really hate all of the below, can we not have any text here? Can we only have auto-generated GitHub release release notes?